dashboards/metrics.jsonnet

```diff
@@ -12,29 +12,29 @@ stdlib.dashboard(
   stdlib.variable('customer', 'Customer', 'label_values(prometheus_tsdb_head_series, customer)')
 )
 .addTemplate(
-  stdlib.variable('control_plane', 'Control Plane', 'label_values(prometheus_tsdb_head_series{customer=~"$customer"}, installation)')
+  stdlib.variable('management_cluster', 'Management Cluster', 'label_values(prometheus_tsdb_head_series{customer=~"$customer"}, installation)')
 )
 
 .addPanel(
   stdlib.singleSeriesChart(
     'Number of Time Series In Prometheus (Total)',
-    'sum(prometheus_tsdb_head_series{customer=~"$customer", cluster_id=~"$control_plane"})',
+    'sum(prometheus_tsdb_head_series{customer=~"$customer", installation=~"$management_cluster"})',
     'Time Series',
   ),
   gridPos={x: 0, y: 0, w: 8, h: 9},
 )
 .addPanel(
   stdlib.multiSeriesChart(
-    'Number of Time Series In Prometheus (Per Control Plane)',
-    'sum(avg(prometheus_tsdb_head_series{customer=~"$customer", cluster_id=~"$control_plane"}) by (customer, cluster_id)) by (cluster_id)',
+    'Number of Time Series In Prometheus (Per Installation)',
+    'sum(avg(prometheus_tsdb_head_series{customer=~".*", installation=~".*"}) by (customer, installation)) by (installation)',
```

Comment:
Doesn't this break templating?

Reply:
Actually, yes. I fixed the templating in another commit